public/app/redux/reducers/continuous/index.ts

```diff
@@ -75,8 +76,14 @@ const initialState: ContinuousState = {
 export const reloadAppNames = createAsyncThunk(
   'names/reloadAppNames',
   async (_, thunkAPI) => {
+    const start = getUTCdate(formatAsOBject(thunkAPI.getState()['continuous'].from), 0);
+    const end = getUTCdate(formatAsOBject(thunkAPI.getState()['continuous'].until), 0);
+
+    const startMs = toUnixTimestamp(start) * 1000;
+    const endMs = toUnixTimestamp(end) * 1000;
```

Comment:
Getting the start/end in milliseconds for a page seems like a pretty common action, but I couldn't find a better way to get this information. Ideally we should extract this out into a function.